assets/javascripts/discourse/components/topic-assignments.gjs

```diff
@@ -41,7 +41,7 @@ export default class TopicAssignments extends Component {
     </div>
     <Assignment
       @assignment={{this.selectedAssignment}}
-      @onSubmit={{this.submit}}
+      @onSubmit={{this.args.onSubmit}}
```

Comment:
```suggestion
      @onSubmit={{@onSubmit}}
```

Reply:
ah yes shit, we don't have linting here 🙃